lib/src/console/console.dart

```diff
@@ -4,16 +4,20 @@ import 'dart:io';
 /// Allows to print log messages and error messages.
 class Console {
   /// Creates an instance with
-  const Console(this._output, this._error);
+  const Console(this._output, this._error, this._supportsAnsiEscapes);
 
-  Console.stdio() : this(stdout, stderr);
+  Console.stdio() : this(stdout, stderr, stdout.supportsAnsiEscapes);
+
+  final bool _supportsAnsiEscapes;
 
   final Stdout _output;
 
   final Stdout _error;
 
   /// Writes the [message] to the error sink
-  void error(Object message) => _error.writeln(message.toString());
+  void error(Object message) => _supportsAnsiEscapes
```

Comment:
Can we change this line to `void error(Object message) => _output.supportsAnsiEscapes` ?